# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1421379911 18000
#      Thu Jan 15 22:45:11 2015 -0500
# Node ID e33427a854ec8bb6269231e84a986daca82a8737
# Parent  6464cd19e0f4083cef9ed4980217f7981d2b9b63
* configure_make.m: Single-quote environment variable values.

diff --git a/scripts/pkg/private/configure_make.m b/scripts/pkg/private/configure_make.m
--- a/scripts/pkg/private/configure_make.m
+++ b/scripts/pkg/private/configure_make.m
@@ -46,17 +46,17 @@ function configure_make (desc, packdir, 
     if (verbose)
       mkoctfile_program = [mkoctfile_program " --verbose"];
     endif
 
     cenv = {"MKOCTFILE"; mkoctfile_program;
             "OCTAVE_CONFIG"; octave_config_program;
             "OCTAVE"; octave_binary;
             "INSTALLDIR"; desc.dir};
-    scenv = sprintf ("%s=\"%s\" ", cenv{:});
+    scenv = sprintf ("%s='%s' ", cenv{:});
 
     ## Configure.
     if (exist (fullfile (src, "configure"), "file"))
       flags = "";
       if (isempty (getenv ("CC")))
         flags = [flags ' CC="' mkoctfile("-p", "CC") '"'];
       endif
       if (isempty (getenv ("CXX")))
